base/uk.ac.stfc.isis.ibex.ui.devicescreens/src/uk/ac/stfc/isis/ibex/ui/devicescreens/list/DeviceScreensTable.java

```diff
@@ -84,6 +85,21 @@ protected String valueFromRow(DeviceDescription row) {
         setSortListener(name.getColumn(), DeviceScreensComparitor.SortedOnType.NAME);
     }
 
+    private void persist() {
+        TableViewerColumn name = createColumn("Saved on server", 3);
+        name.setLabelProvider(new DataboundCellLabelProvider<DeviceDescription>(observeProperty("persist")) {
+            @Override
+            protected String valueFromRow(DeviceDescription row) {
+                if (row.getPersist()) {
+                    return "Yes";
+                } else {
+                    return "No";
```

Comment:
This would be easier to read at a glance if "Yes" was a checkmark and "No" was blank